CMakeModules/FetchMbedTLS.cmake

```diff
@@ -22,10 +22,28 @@ include(FetchContent)
 
 FetchContent_Declare(
   mbedtls
-  PATCH_COMMAND git apply "${CMAKE_CURRENT_LIST_DIR}/mbedtls.patch"
   GIT_REPOSITORY http://github.com/mbed-TLS/mbedtls.git
   GIT_TAG        v3.6.3.1
   GIT_SHALLOW    1
 )
 
+get_property(
+    compile_options
+    DIRECTORY
+    PROPERTY COMPILE_OPTIONS
+)
+
+set_property(
+    DIRECTORY
+    APPEND
+    PROPERTY COMPILE_OPTIONS -Wno-unterminated-string-initialization
```

Comment:
Is this also supported by clang, or is it a gcc only option? I am not finding it in clang documentation.